package.json

```diff
@@ -96,6 +96,7 @@
 		"@babel/preset-env": "7.8.4",
 		"@babel/register": "7.7.7",
 		"@wordpress/base-styles": "1.5.0",
+		"@wordpress/blocks": "6.19.0",
```

Comment:
We don't typically need to bundle those packages since they're provided by WordPress itself already. I haven't tested this PR yet, but at first look I don't think this should be necessary here.

Reply:
Hey guys, thanks for the comment. I'll double-check this :+1:

Reply:
> Because the getCategoryWithFallbacks helper will be needed in a couple of other repositories, what do you think about turning it into an npm module (placed inside Calypso packages)?

Yeah, ideally we'd have this in a common place shared among all packages that need it (no only calypso-related. This [came](https://github.com/Automattic/wp-calypso/pull/43670#discussion_r445732682) to my mind last week and I thought about adding it [here](https://github.com/WordPress/gutenberg/pull/23695). If you still think it's be better to put it under Calypso package, let me know.

Reply:
> One more code comment here, cause I couldn't leave it on an empty file - Looks like the docker/wordpress/.gitkeep file got here by an accident.

Woopsie, yes, thanks for catching that :)

Reply:
>     We don't typically need to bundle those packages since they're provided by WordPress itself already. I haven't tested this PR yet, but at first look I don't think this should be necessary here

I've double-checked and indeed it wasn't needed. I've removed it :+1:

Reply:
@WunderBart I just noticed that the `.gitkeep` was already there, I just accidentally changed its permissions by accident. I reverted my accidental changes.